src/tool_use/scripts/activity_tracker.py

```diff
@@ -0,0 +1,609 @@
+from pathlib import Path
+import sqlite3
+import time
+import datetime
+from typing import Optional, Tuple, List
+from rich.console import Console
+from rich.table import Table
+from ..utils.ai_service import AIService
+
+HELP_TEXT = """Usage: ai log [command] [args]
+
+Commands:
+  <activity>           Start tracking an activity
+  (no arguments)       Stop current activity or prompt to start one
+  tell <query>         Query your activity history in natural language
+  category <command>   Manage activity categories
+  help                Show this help message
+
+Examples:
+  ai log working on python project
+  ai log tell me how long I coded today
+  ai log tell me what I did yesterday
+  ai log category list
+
+For more details on a command, use: ai log <command> help
+"""
+
+CATEGORY_HELP_TEXT = """Usage: ai log category <command> [args]
+
+Commands:
+  list              List all categories with usage counts
+  rename <old> <new> Rename a category
+  merge <from> <into> Merge one category into another
+  show <name>       List activities in a category
+  help              Show this help message
+
+Examples:
+  ai log category list
+  ai log category rename "Coding" "Programming"
+  ai log category merge "Dev" "Programming"
+  ai log category show "Programming"
+"""
+
+
```

Comment:
directory/config stuff should probably be extracted to a utility at some point, not sure if it's worthwhile to do now